qmxgraph/widget.py

```diff
@@ -333,18 +359,31 @@ def is_enabled(self) -> bool:
 
     def _connect_events_bridge(self):
 
-        self.api.register_cells_added_handler('bridge_events_handler.cells_added_slot', check_api=False)
-        self.api.register_cells_removed_handler('bridge_events_handler.cells_removed_slot', check_api=False)
-        self.api.register_label_changed_handler('bridge_events_handler.label_changed_slot', check_api=False)
+        self.api.register_cells_added_handler(
+            'bridge_events_handler.cells_added_slot', check_api=False
+        )
+        self.api.register_cells_removed_handler(
+            'bridge_events_handler.cells_removed_slot', check_api=False
+        )
+        self.api.register_label_changed_handler(
+            'bridge_events_handler.label_changed_slot', check_api=False
+        )
         self.api.register_selection_changed_handler(
-            'bridge_events_handler.selection_changed_slot', check_api=False)
+            'bridge_events_handler.selection_changed_slot', check_api=False
+        )
         self.api.register_terminal_changed_handler(
-            'bridge_events_handler.terminal_changed_slot', check_api=False)
+            'bridge_events_handler.terminal_changed_slot', check_api=False
+        )
         self.api.register_terminal_with_port_changed_handler(
-            'bridge_events_handler.terminal_with_port_changed_slot', check_api=False)
+            'bridge_events_handler.terminal_with_port_changed_slot', check_api=False
+        )
         self.api.register_cells_bounds_changed_handler(
-            'bridge_events_handler.cells_bounds_changed_slot', check_api=False)
-        self.api.register_view_update_handler('bridge_events_handler.view_update_slot', check_api=False)
+            'bridge_events_handler.cells_bounds_changed_slot', check_api=False
+        )
+        self.api.register_view_update_handler(
+            'bridge_events_handler.view_update_slot', check_api=False
+        )
+
     #
     # def set_double_click_handler(self, handler):
```

Comment:
I think this wasn't removed in the other PR for "removing dead code".

Reply:
Ahh right, thanks. Found a bunch of other code here that was commented out but was not marked with the task (that's why I missed it).